src/wtsdk/src/cj4/ui/fmc/CJ4_FMC_Page.ts

```diff
@@ -0,0 +1,42 @@
+export abstract class CJ4_FMC_Page {

```

Comment:
I wonder if we could make this an interface instead and leave the internals and implementation to the specific pages.

Reply:
while interface is always good too ofc. i want to provide the pages with the 90% case base implementation... ?